extensions/notebook/package.nls.json

```diff
@@ -6,6 +6,7 @@
 	"notebook.useExistingPython.description": "Local path to a preexisting python installation used by Notebooks.",
 	"notebook.overrideEditorTheming.description": "Override editor default settings in the Notebook editor. Settings include background color, current line color and border",
 	"notebook.maxTableRows.description": "Maximum number of rows returned per table in the Notebook editor",
+	"notebook.maxBookSearchDepth.description": "Maximum number of subdirectories to search for Books",
```

Comment:
I think this should be "Maximum depth of subdirectories to search for Books" as it might read that you only check 5 folders under the root instead.

Reply:
We should also define what to use if you want infinite depth. For example is 0 correct in that case?

Reply:
Will use 0 for infinite.